float32 eyeLeftX
float32 eyeLeftY
float32 eyeRightX
float32 eyeRightY
float32 noseX
float32 noseY
float32 mouthCornerLeftX
float32 mouthCornerLeftY
float32 mouthCornerRightX
float32 mouthCornerRightY
